vic/drivers/classic/src/read_initial_model_state.c

```diff
@@ -42,29 +42,33 @@ read_initial_model_state(FILE            *init_state,
                          int              Nveg,
                          int              Nbands,
                          int              cellnum,
-                         soil_con_struct *soil_con)
+                         soil_con_struct *soil_con,
+                         lake_con_struct  lake_con)
 {
-    extern option_struct options;
+    extern option_struct     options;
```

Comment:
This is aligned with the section of variables below.  No need to make further edits.